services/ui/serverless.yml

```diff
@@ -33,9 +34,19 @@ custom:
   route53DomainName: ${ssm:/configuration/${self:custom.stage}/route53/domainName~true, ""}
   cloudfrontCertificateArn: ${ssm:/configuration/${self:custom.stage}/cloudfront/certificateArn~true, ssm:/configuration/default/cloudfront/certificateArn~true, ""}
   cloudfrontDomainName: ${ssm:/configuration/${self:custom.stage}/cloudfront/domainName~true, ""}
-  wafLoggingBucket: ${cf:ui-waflog-s3-bucket-${self:custom.stage}.WaflogsUploadBucketArn}
   iamPath: ${ssm:/configuration/${self:custom.stage}/iam/path~true, ssm:/configuration/default/iam/path~true, "/"}
   iamPermissionsBoundaryPolicy: ${ssm:/configuration/${self:custom.stage}/iam/permissionsBoundaryPolicy~true, ssm:/configuration/default/iam/permissionsBoundaryPolicy~true, ""}
+  webAclName: ${self:service}-${self:custom.stage}-allow-usa-plus-territories
+  firehoseStreamName: aws-waf-logs-${self:service}-${self:custom.stage}-firehose
+  scripts:
+    hooks:
+      # Associate the WAF ACL with the Firehose Delivery Stream
+      deploy:finalize: |
```

Comment:
This hook associates the WAF web acl with the Firehose delivery stream.  This code largely came from the ui-waf-log-assoc service, which is removed in this PR.  With a few more lookups to get some ARNs, this can run in the same service as ui.  For speed and maintainability sake, that service was removed and its logic put here.